### PR TITLE
Workflows

### PR DESCRIPTION
**Pull request by addy419**

### CODE REVIEW

**Comment by addy419** [comment]
I've removed the timeout option from SubmissionResult because it's already available in IndicoConfig

File: examples/WorkflowSubmission.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import org.json.JSONObject;
+
+public class WorkflowSubmission {
+
+    public static void main(String args[]) throws IOException {
+        IndicoConfig config = new IndicoConfig.Builder()
+                .host("dev.indico.io")
```
Comment (nkassis) [inline examples/WorkflowSubmission.java:21]: lets change this to app.indico.io dev not technically public

File: src/main/graphql/WorkflowJobGraphQL.graphql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+mutation WorkflowJobGraphQL($workflowId: Int!, $files: [FileInput]!) {
```
Comment (nkassis) [inline src/main/graphql/WorkflowJobGraphQL.graphql:1]: I had to make a modification to the python client due to a bug, we were missing a flag here. Here the PR: https://github.com/IndicoDataSolutions/indico-client-python/pull/66/files

File: examples/WorkflowSubmission.java (hunk 1 above)
Comment (addy419) [reply examples/WorkflowSubmission.java]: Fixed

File: src/main/graphql/WorkflowJobGraphQL.graphql (hunk 2 above)
Comment (addy419) [reply src/main/graphql/WorkflowJobGraphQL.graphql]: Fixed

**Pull request merged**